README.md

```diff
@@ -0,0 +1,3 @@
+# Running as a Service:
```

Comment:
Well, using the Task Scheduler is indeed a valid solution. Note though that services and scheduled tasks are different things in Windows, so fix the title please. Also, the `:` is redundant.